docs/user-manuals/jobsidecarterminator.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Job Sidecar Terminator
+---
+
+**FEATURE STATE:** Kruise v1.4.0
+
+In Kubernetes world, we hope that Pods controlled by job-type workload enter `Completed` phase immediately once the task executed in main container is completed.
+However, in the scene where the these Pods have long-running sidecar containers, Pod may be out of our expectations because the long-running sidecar container cannot be terminated when the main containers completed.
+
+To solve such problem, we proposal a controller, namely sidecar terminator, to watch and terminate sidecar containers for such job-type Pods when its main containers completed.
+
+## Requirements
+
+ - Enabled `JobSidecarTerminator` feature gate when installing/upgrading Kruise (defaults to `disabled`).
+ - Enabled `KruiseDaemon` feature gate when installing/upgrading Kruise (defaults to `enabled`).
```

Comment:
comments about how the siderterminator behaves in serverless container such Fargate